src/docs/clients/python/api.mdx

```diff
@@ -23,21 +23,21 @@ This gives you an overview of the public API that raven-python exposes.
 
 : This method is the low-level method for reporting events to Sentry. It captures and processes an event and pipes it via the configured transport to Sentry.
 
-  Example:
-
-  ```python
-  capture('raven.events.Message', message='foo', data={
-      'request': {
-          'url': '...',
-          'data': {},
-          'query_string': '...',
-          'method': 'POST',
-      },
-      'logger': 'logger.name',
-  }, extra={
-      'key': 'value',
-  })
-  ```
+Example:
```

Comment:
Definition lists don't support nesting like this it appears. if you look at what the production docs look like for the `clients/python/api` https://docs.sentry.io/clients/python/api/ you'll see it doesn't nest either.